### PR TITLE
[CAMEL-18159] SendDynamicAware of several components parses uri that starts with schema:// incorectly

### PR DESCRIPTION
**Pull request by artemse**

### CODE REVIEW

File: components/camel-ftp/src/test/java/org/apache/camel/component/file/remote/FtpSendDynamicAwareTest.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package org.apache.camel.component.file.remote;
```
Comment (davsclaus) [inline components/camel-ftp/src/test/java/org/apache/camel/component/file/remote/FtpSendDynamicAwareTest.java:1]: This class looks like its not complete, or added in the git PR as a mistake

Comment (davsclaus) [inline components/camel-ftp/src/test/java/org/apache/camel/component/file/remote/FtpSendDynamicAwareTest.java:1]: This class looks like its not complete, or added in the git PR as a mistake

Comment (artemse) [reply components/camel-ftp/src/test/java/org/apache/camel/component/file/remote/FtpSendDynamicAwareTest.java]: Yes, the file was added by mistake. Removed it.

File: components/camel-rabbitmq/src/test/java/org/apache/camel/component/rabbitmq/RabbitMQSendDynamicAwareTest.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
+package org.apache.camel.component.rabbitmq;
```
Comment (davsclaus) [inline components/camel-rabbitmq/src/test/java/org/apache/camel/component/rabbitmq/RabbitMQSendDynamicAwareTest.java:1]: You need to add the ASF license header in top of these new files. Look at one of the existing java source files and copy its header.

**Review by davsclaus — APPROVED** [review]
ASF license headers is missing in some files

**Comment by artemse** [comment]
I believe it's last component with this problem.

**Review by davsclaus — APPROVED** [review]

**Pull request merged**